BoxContentSDK/BoxContentSDK/Models/BOXRecentItem.h

```diff
@@ -0,0 +1,41 @@
+//
+//  BOXRecentItem.h
+//  Pods
+//
+//  Created by Andrew Dempsey on 12/19/16.
+//
+//
+
+#import <BoxContentSDK/BoxContentSDK.h>
+
+/**
+ Represents an item recently interacted with by the current user.
+ */
+@interface BOXRecentItem : BOXModel
+
+/**
+ The item that was recently interacted with by the user.
```

Comment:
Should we document our expectation of it being a file or bookmark at this level?